src/classes/ADDR_Contact_TEST.cls

```diff
@@ -0,0 +1,1112 @@
+/*
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.